lib/kube/proxy/sess.go

```diff
@@ -612,21 +615,32 @@ func (s *session) launch() error {
 
 	s.io.On()
 	if err = executor.StreamWithContext(s.streamContext, options); err != nil {
+		s.setTerminationErr(err)
 		s.reportErrorToSessionRecorder(err)
 		s.log.WithError(err).Warning("Executor failed while streaming.")
+
 		return trace.Wrap(err)
 	}
 	return nil
 }
 
+func (s *session) setTerminationErr(err error) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	if s.terminationErr != nil {
+		return
```

Comment:
Maybe debug log second error?

Reply:
Not necessary. This happens all the time because StreamWithContext will return an error when the session context is cancelled.